src/game/savegame/savegame_bson.c

```diff
@@ -812,16 +812,13 @@ static bool SaveGame_BSON_LoadCurrentMusic(struct json_object_s *music_obj)
     }
 
     int16_t current_track = json_object_get_int(music_obj, "current_track", -1);
-    int32_t timestamp_arr[2];
-    timestamp_arr[0] = json_object_get_int(music_obj, "timestamp1", 0);
-    timestamp_arr[1] = json_object_get_int(music_obj, "timestamp2", 0);
-    int64_t *timestamp = (int64_t *)timestamp_arr;
+    int64_t timestamp = json_object_get_int64(music_obj, "timestamp", -1);
     if (current_track) {
         Music_Play(current_track);
-        if (!Music_SeekTimestamp(current_track, *timestamp)) {
+        if (!Music_SeekTimestamp(current_track, timestamp)) {
             LOG_WARNING(
                 "Could not load current track %d at timestamp %d.",
-                current_track, *timestamp);
+                current_track, timestamp);
```

Comment:
Will %d work with int64_t?

Reply:
Here's what I kept reading on stack overflow

https://stackoverflow.com/questions/9225567/how-to-portably-print-a-int64-t-type-in-c

Reply:
Oh oops I missed that last night and my early morning brain did not read your comment correctly lol. Fixed.